app/src/Blocks/Column/MultiColumnBlock.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+
+namespace SilverStripe\Bambusa\Blocks\Column;
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\GridField\GridFieldAddNewButton;
+use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
+use SilverStripe\Shared\Models\ContentItem;
+use Symbiote\GridFieldExtensions\GridFieldOrderableRows;
+
+class MultiColumnBlock extends BaseElement
```

Comment:
Maybe this should be in a separate module?

Reply:
Good thought, but I've mentioned in backlog planning that it's not a priority for this one. The color and font picker were effectively already module (in cwp-agencyextensions), just not isolated out too well from other stuff.

Reply:
Yes, however, I assume that should have been decided in the backlog planning. Unfortunately, I wasn't able to attend the one, but my assumption from the ACs and 2 pointer estimate is we decided not to build a completely new module in case we couldn't find something reusable and maintained by community.